spec/lib/data_update_scripts/nullify_empty_tag_colors_spec.rb

```diff
@@ -0,0 +1,30 @@
+require "rails_helper"
+require Rails.root.join(
+  "lib/data_update_scripts/20211112173720_nullify_empty_tag_colors.rb",
+)
+
+describe DataUpdateScripts::NullifyEmptyTagColors do
+  let(:empty_bg_tag) { create(:tag) }
+  let(:empty_fg_tag) { create(:tag) }
+
+  before do
+    empty_bg_tag.update_columns(bg_color_hex: "")
+    empty_fg_tag.update_columns(text_color_hex: "")
+  end
+
+  it "sets empty string in background color to nil" do
```

Comment:
_**Quick, non-blocking suggestion:**_ should we use `aggregate_failures` on this test and the one below it since they have multiple expectations?
```suggestion
  it "sets empty string in background color to nil", :aggregate_failures do
```

Reply:
I believe I can modify this to be a single expectation rather than "assert precondition, exercise, assert postcondition" - and I'll fix that up - thanks for the recommendation, yes, if there were actually multiple expectation aggregation would make sense
  
Change from

```ruby
expect(state).to eq(a)
do_it!
expect(state).to eq(b)
```
to

```ruby
expect { do_it! }.to change(state).from(a).to(b)
```